django_project/core/models/item/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from core.models.metadata.models import Metadata
+
+
+class Item(Metadata):
+    primary_name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class ItemRating(Metadata):
+    item = models.ForeignKey(Item, on_delete=models.CASCADE)
+    rating = models.ForeignKey('Rating', on_delete=models.CASCADE)
+    # region = models.ForeignKey('Region', on_delete=models.CASCADE)
+    collection_type = models.ForeignKey('CollectionType', on_delete=models.CASCADE)  # if blank, assume rating applies to all of 'item'
+    collection_method = models.ForeignKey('CollectionMethod', blank=True, null=True, on_delete=models.CASCADE)  # if blank, assume rating applies to all of 'collection_type'; if 'collection_type' is blank, this must also be blank
+
+
+# class AlternativeName(Metadata):
```

Comment:
Did you mean to put commented code in the PR?

Reply:
Usually I wouldn't, but in this case I wanted to hint at the direction we're headed :)